apps/cli/src/index.tsx

```diff
@@ -95,38 +336,538 @@ const App: React.FC = () => {
+      }
+    });
+    if (streamingPhase !== "awaitingFeedback") return null;
     return (
-      <Box flexDirection="column" height={"100%"}>
-        <TerminalInterface
-          message={message}
-          setMessage={() => setMessage(null)}
-          CustomInput={CustomInput}
-        />
-        <Box flexGrow={1} />
-        <Box width="100%" justifyContent="flex-end">
-          <Text color="gray" dimColor>
-            logged in
-          </Text>
-        </Box>
+      <Box>
+        <Text>Plan feedback (approve/deny): {input}</Text>
+      </Box>
+    );
+  };
+
+  // Streaming logic: when prompt is set, stream logs
+  useEffect(() => {
+    if (prompt && selectedRepo && streamingPhase === "streaming") {
+      setLogs([]);
+      setPlannerFeedback(null);
+      setLoadingLogs(true);
+      (async () => {
+        try {
+          const userAccessToken = getAccessToken();
+          const installationAccessToken = await getInstallationAccessToken();
+          const encryptionKey = process.env.SECRETS_ENCRYPTION_KEY;
+          if (!userAccessToken || !installationAccessToken || !encryptionKey) {
+            setLogs([
+              `Missing secrets: ${userAccessToken ? "" : "userAccessToken, "}${installationAccessToken ? "" : "installationAccessToken, "}${encryptionKey ? "" : "encryptionKey"}`,
+            ]);
+            return;
+          }
+          const encryptedUserToken = encryptSecret(
+            userAccessToken,
+            encryptionKey,
+          );
+          const encryptedInstallationToken = encryptSecret(
+            installationAccessToken,
+            encryptionKey,
+          );
+          const [owner, repoName] = selectedRepo.full_name.split("/");
+          const runInput = {
+            messages: [
+              {
+                id: uuidv4(),
+                type: "human",
+                content: [{ type: "text", text: prompt }],
+              },
+            ],
+            targetRepository: {
+              owner,
+              repo: repoName,
+              branch: selectedRepo.default_branch || "main",
+            },
+            autoAcceptPlan: false,
+          };
+          const installationId = getInstallationId();
+          const newClient = new Client({
+            apiUrl: LANGGRAPH_URL,
+            defaultHeaders: {
+              GITHUB_TOKEN_COOKIE: encryptedUserToken,
+              GITHUB_INSTALLATION_TOKEN_COOKIE: encryptedInstallationToken,
+              GITHUB_INSTALLATION_NAME: owner,
+              GITHUB_INSTALLATION_ID: installationId,
+            },
+          });
+          setClient(newClient);
+          const thread = await newClient.threads.create();
+          const threadId = thread.thread_id;
+          setThreadId(threadId);
+          const run = await newClient.runs.create(threadId, MANAGER_GRAPH_ID, {
+            input: runInput,
+            config: { recursion_limit: 400 },
+            ifNotExists: "create",
```

Comment:
if doing `ifNotExists: create`, just create a new UUID and send that
```suggestion
          const threadId = uuidv4();
          setThreadId(threadId);
          const run = await newClient.runs.create(threadId, MANAGER_GRAPH_ID, {
            input: runInput,
            config: { recursion_limit: 400 },
            ifNotExists: "create",
```

Reply:
i removed if not exists: create!